package/sbp_rtcm3_bridge/src/rtcm3_decode.c

```diff
@@ -935,4 +985,11 @@ void init_data(rtcm_sat_data *sat_data) {
   for (u8 freq = 0; freq < NUM_FREQS; ++freq) {
     sat_data->obs[freq].flags.data = 0;
   }
-}
\ No newline at end of file
+}
+
+void update_pseudorange_ambiguity(gnss_signal16_t sid, gps_time_nano_t tor, double pseudorange){
+  if( sid.sat >= 0 && sid.sat <= RTCM_MAX_SATS) {
+    pseudorange_amb[sid.sat].amb = (u8)(pseudorange / PRUNIT_GPS);
```

Comment:
Technically, I think this should be floor() rather than (u8), but since `pseudorange` hopefully won't be negative, I don't think it's a problem.